# THE SPEARMAN RANK CORRELATION COEFFICIENT

lso known as Spearman's rho or Spearman's rank correlation coefficient, is a statistical measure that assesses the strength and direction of the monotonic relationship between two variables. It is an alternative to the Pearson correlation coefficient, which measures the linear relationship between variables.

#### 13.10.1

The following table shows 15 randomly selected geographic areas ranked by population density and
age-adjusted death rate. Can we conclude at the .05 level of significance that population density and
age-adjusted death rate are not mutually independent?

In [1]:
data <- read.csv("data/ch13_all/EXR_C13_S10_01.csv")
head(data)
attach(data)

SUBJ X  Y 
1 1     8 10
2 2     2 14
3 3    12  4
4 4     4 15
5 5     9 11
6 6     3  1

In [2]:
cor.test(X,Y,method = "spearman")


	Spearman's rank correlation rho

data:  X and Y
S = 600, p-value = 0.8025
alternative hypothesis: true rho is not equal to 0
sample estimates:
        rho 
-0.07142857 


Based on the analysis using Spearman's rank correlation coefficient, with a calculated value of -0.071 and a p-value of 0.8025, we fail to reject the null hypothesis at the 0.05 level of significance. Therefore, we cannot conclude that there is a significant association between population density and age-adjusted death rate. The results suggest that these two variables are likely to be mutually independent.

#### 13.10.2

The following table shows 10 communities ranked by decayed, missing, or filled (DMF) teeth per 100
children and fluoride concentration in ppm in the public water supply.

Do these data provide sufficient evidence to indicate that the number of DMF teeth per 100 children
tends to decrease as fluoride concentration increases? Let $\alpha$ = .05.

In [3]:
data <- read.csv("data/ch13_all/EXR_C13_S10_02.csv")
head(data)
attach(data)

SUBJ X Y
1 1    8 1
2 2    9 3
3 3    7 4
4 4    3 9
5 5    2 8
6 6    4 7

The following objects are masked from data (pos = 3):

    SUBJ, X, Y




In [4]:
cor.test(X,Y,method = "spearman")


	Spearman's rank correlation rho

data:  X and Y
S = 322, p-value < 2.2e-16
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
-0.9515152 


Based on the analysis using Spearman's rank correlation coefficient, with a calculated value of -0.952 and an extremely small p-value (< 2.2e-16), we can conclude that there is strong evidence to support the claim that the number of decayed, missing, or filled (DMF) teeth per 100 children tends to decrease as fluoride concentration in the public water supply increases. The negative correlation coefficient suggests a strong negative monotonic relationship between the two variables. Therefore, higher fluoride concentrations are associated with lower rates of DMF teeth in the communities examined.

#### 13.10.3

The purpose of a study by Nozawa et al. was to evaluate the outcome of surgical repair of pars
interarticularis defect by segmental wire fixation in young adults with lumbar spondylolysis. The
authors cite literature indicating that segmental wire fixation has been successful in the treatment of nonathletes with spondylolysis and point out that no information existed on the results of this type of surgery in athletes. In a retrospective study of subjects having surgery between 1993 and 2000, the authors found 20 subjects who had undergone the surgery. The following table shows the age (years) at surgery and duration (months) of follow-up care for these subjects.

May we conclude, on the basis of these data, that in a population of similar subjects there is an
association between age and duration of follow-up? Let $\alpha$ = .05.

In [5]:
data <-read.csv("data/ch13_all/EXR_C13_S10_03.csv")
head(data,3)
attach(data)

FOLLOWUP AGE
1 103      37 
2  68      27 
3  62      12

In [6]:
cor.test(FOLLOWUP, AGE, method = "spearman")

Warning message in cor.test.default(FOLLOWUP, AGE, method = "spearman"):
“Cannot compute exact p-value with ties”



	Spearman's rank correlation rho

data:  FOLLOWUP and AGE
S = 1305.8, p-value = 0.9392
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
0.01823019 


Based on the analysis using Spearman's rank correlation coefficient, with a calculated value of 0.018 and a p-value of 0.9392, we fail to reject the null hypothesis at the 0.05 level of significance. Therefore, we cannot conclude that there is a significant association between age at surgery and the duration of follow-up in a population of similar subjects. The results suggest that there is no clear monotonic relationship between these two variables.

#### 13.10.4

Nozawa et al. also calculated the Japanese Orthopaedic Association score for measuring back pain (JOA). The results for the 20 subjects along with the duration of follow-up are shown in the following table. The higher the number, the lesser the degree of pain.
Can we conclude from these data that in general there is a relationship between length of follow-up
and JOA score at the time of the operation? Let $\alpha$ = .05.

In [7]:
data <-read.csv("data/ch13_all/EXR_C13_S10_04.csv")
head(data,3)
attach(data)

MONTHS JOA
1 103    21 
2  68    14 
3  62    26

In [8]:
cor.test(MONTHS, JOA, method = "spearman")

Warning message in cor.test.default(MONTHS, JOA, method = "spearman"):
“Cannot compute exact p-value with ties”



	Spearman's rank correlation rho

data:  MONTHS and JOA
S = 1469.3, p-value = 0.6603
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
-0.1047632 


Based on the analysis using Spearman's rank correlation coefficient, with a calculated value of -0.105 and a p-value of 0.6603, we fail to reject the null hypothesis at the 0.05 level of significance. Therefore, we cannot conclude that there is a significant relationship between the length of follow-up and the Japanese Orthopaedic Association (JOA) score at the time of the operation in general. The results suggest that there is no clear monotonic relationship between these two variables.

#### 13.10.5

Butz et al. studied the use of noninvasive positive-pressure ventilation by patients with
amyotrophic lateral sclerosis. They evaluated the benefit of the procedure on patients’ symptoms,
quality of life, and survival. Two variables of interest are PaCO2, partial pressure of arterial carbon dioxide, and PaO2, partial pressure of arterial oxygen. The following table shows, for 30 subjects, values of these variables (mm Hg) obtained from baseline arterial blood gas analyses.

On the basis of these data may we conclude that there is an association between PaCO2 and PaO2
values? Let $\alpha$ = .05.

In [9]:
data <-read.csv("data/ch13_all/EXR_C13_S10_05.csv")
head(data,3)
attach(data)

PACO2 PAO2
1 40    101 
2 47     69 
3 34    132

In [10]:
cor.test(PACO2, PAO2, method = "spearman")

Warning message in cor.test.default(PACO2, PAO2, method = "spearman"):
“Cannot compute exact p-value with ties”



	Spearman's rank correlation rho

data:  PACO2 and PAO2
S = 6430.6, p-value = 0.01753
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
-0.430622 


Based on the analysis using Spearman's rank correlation coefficient, with a calculated value of -0.431 and a p-value of 0.01753, we can conclude that there is a significant association between PaCO2 and PaO2 values in the population of patients with amyotrophic lateral sclerosis. The negative correlation coefficient suggests an inverse relationship between these two variables. As PaCO2 increases, PaO2 tends to decrease. Therefore, changes in PaCO2 levels are associated with changes in PaO2 levels in this context.

#### 13.10.6

Seventeen patients with a history of congestive heart failure participated in a study to assess the
effects of exercise on various bodily functions. During a period of exercise the following data were
collected on the percent change in plasma norepinephrine (Y) and the percent change in oxygen
consumption (X):

On the basis of these data can one conclude that there is an association between the two variables? Let $\alpha$ = .05.

In [11]:
data <-read.csv("data/ch13_all/EXR_C13_S10_06.csv")
head(data,3)
attach(data)

SUBJECT X   Y  
1 1       500 525
2 2       475 130
3 3       390 325

The following objects are masked from data (pos = 6):

    X, Y


The following objects are masked from data (pos = 7):

    X, Y




In [12]:
cor.test(X, Y, method = "spearman")

Warning message in cor.test.default(X, Y, method = "spearman"):
“Cannot compute exact p-value with ties”



	Spearman's rank correlation rho

data:  X and Y
S = 246.95, p-value = 0.00186
alternative hypothesis: true rho is not equal to 0
sample estimates:
      rho 
0.6973605 


Based on the analysis using Spearman's rank correlation coefficient, with a calculated value of 0.697 and a p-value of 0.00186, we can conclude that there is a significant association between the percent change in plasma norepinephrine and the percent change in oxygen consumption in patients with a history of congestive heart failure. The positive correlation coefficient suggests a positive relationship between these two variables. As the percent change in oxygen consumption increases, the percent change in plasma norepinephrine also tends to increase. Therefore, changes in oxygen consumption are associated with changes in plasma norepinephrine levels in this population.